components/suggestion/command_provider.jsx

```diff
@@ -54,6 +54,17 @@ export class CommandSuggestion extends Suggestion {
 }
 
 export default class CommandProvider extends Provider {
+    constructor(props) {
```

Comment:
Maybe don't call them props. I thought this was a component for a second.

Reply:
@crspeller Sure, but I can't think of anything better. I'll follow your suggestion. 
(I was following https://github.com/mattermost/mattermost-webapp/blob/fca8b79c231b001aca4300846c621c564c15cbb7/components/suggestion/at_mention_provider/at_mention_provider.jsx#L18)

Reply:
You could just de-structure them, so you don't have to name them. 
But if it's consistent with other places then I think it's probably fine.

Reply:
@crspeller Yep, good idea, done. And I simplified it. No need to call a setProps function--the code is simple enough that people will know what's happening.